pdns/ws-api.cc

```diff
@@ -188,7 +189,7 @@ void apiServerStatistics(HttpRequest* req, HttpResponse* resp) {
 
     doc.push_back(Json::object {
       { "type", "MapStatisticItem" },
-      { "name", "queries-by-qtype" },
+      { "name", "response-by-qtype" },
       { "value", values },
```

Comment:
Perhaps we could add a new `response-by-qtype` field while deprecating the existing `queries-by-qtype` one in 4.2.0, so it doesn't break existing setups right away? It probably doesn't matter much, though.

Reply:
Is it needed to deprecate the existing `queries-by-qtype` ? This value wasn't on the last stable 4.1.5 release. Maybe just informing the alpha1 users that this value changes ?
But if we need to deprecate it, what's the best way to do ?

Reply:
Sorry, I had in mind that this metric already existed in 4.1.x but I was wrong. Please forget that comment :-)